logging.go

```diff
@@ -147,3 +147,60 @@ func (lr *LoggingResource) ValidateAWSTrustPolicy(ctx context.Context, awsExtern
 	}
 	return lr.do(req, nil)
 }
+
+// NetworkFlowLog represents a network flow log entry from the Tailscale API.
+type NetworkFlowLog struct {
+	Logged          string         `json:"logged"`
```

Comment:
Can we make Logged, Start, and End into `time.Time` values? Among other things, that will make it more clear that they denote timestamps, which we'd otherwise need to document somewhere.

(I realize the API docs say that, but the user of this library package will be seeing the docs on pkg.go.dev, and ideally shouldn't have to go dig up the API docs to figure out what these fields denote)